server.js

```diff
@@ -254,6 +254,36 @@ app.get("/task/cash", (req, res) => {
     }
 });
 
+setInterval(()=>{
```

Comment:
Consider using throttling or debouncing in favor of setInterval. The interval is really overloaded with a lot of functions, and each function would have to be managed individually, if an error were to occur. 

https://lodash.com/docs/4.17.11#throttle
https://lodash.com/docs/4.17.11#debounce